test/unit/math/prim/fun/pow_test.cpp

```diff
@@ -4,18 +4,16 @@
 #include <cmath>
 
 TEST(MathFunctions, pow) {
-  using stan::math::pow;
-  using std::pow;
-
-  EXPECT_FLOAT_EQ(pow(2.15, 1.71), 3.70228430936580996892756228068688);
-  EXPECT_FLOAT_EQ(pow(7.62, -1.15), 0.0967724604023293179518727243992);
+  EXPECT_FLOAT_EQ(stan::math::pow(2.15, 1.71),
```

Comment:
I don't know if these show up elsewhere, but we need to test integer and double values.  And we can test them against results return by casting them up to double and calling the std versions.